### PR TITLE
Breaking changes updates

### PR DESCRIPTION
**Pull request by sfshaza2**
Staged: https://sz-flutter.firebaseapp.com/docs/release/breaking-changes

FYI, @kwalrath, I am not as strict about Google Style guidelines with the breaking change pages.

### CODE REVIEW

File: src/docs/release/breaking-changes/android-activity-control-surface-attach.md [hunk 1]
```diff
@@ -94,30 +105,36 @@ void attachToActivity(
...
 ## Timeline
 
-Landed in version: https://github.com/flutter/engine/pull/21272<br>
+Landed in version: 1.23.0-7.0.pre
 In stable release: not yet
 
 ## References
 
-Motivating bug: https://github.com/flutter/flutter/issues/66192.
+Motivating bug: [Issue #66192][]&emdash;Non exclusive
```
Comment (kwalrath) [inline src/docs/release/breaking-changes/android-activity-control-surface-attach.md:127]:
```suggestion
Motivating bug: [Issue #66192][]—Non exclusive
```

File: src/docs/release/breaking-changes/android-activity-control-surface-attach.md [hunk 2]
```diff
@@ -94,30 +105,36 @@ void attachToActivity(
...
-you perform when the activity is being naturally destroyed.
+to handle your activity being replaced by another
+activity being attached to the [`FlutterEngine`][].
+Generally, you want to perform the same detaching operations
+as performed when the activity is being naturally destroyed.
 
 ## Timeline
 
-Landed in version: https://github.com/flutter/engine/pull/21272<br>
+Landed in version: 1.23.0-7.0.pre
```
Comment (kwalrath) [inline src/docs/release/breaking-changes/android-activity-control-surface-attach.md:122]:
```suggestion
Landed in version: 1.23.0-7.0.pre<br>
```

File: src/docs/release/breaking-changes/android-v1-embedding-create-deprecation.md [hunk 3]
```diff
@@ -1,66 +1,77 @@
...
+Since the v2 embeddings were strongly established over
+the 7 months since the launch of Flutter v1.12,
+we disabled the creation of new app and plugin
 projects using the v1 embeddings.
 
 [add-to-app]: /docs/development/add-to-app
-[io.flutter.embedding]: https://cs.opensource.google/flutter/engine/+/master:shell/platform/android/io/flutter/embedding/
-[io.flutter.app]: https://cs.opensource.google/flutter/engine/+/master:shell/platform/android/io/flutter/app/.
+[`io.flutter.embedding`]: https://cs.opensource.google/flutter/engine/+/master:shell/platform/android/io/flutter/embedding/
+[`io.flutter.app]`: https://cs.opensource.google/flutter/engine/+/master:shell/platform/android/io/flutter/app/.
```
Comment (kwalrath) [inline src/docs/release/breaking-changes/android-v1-embedding-create-deprecation.md:46]:
```suggestion
[`io.flutter.app`]: https://cs.opensource.google/flutter/engine/+/master:shell/platform/android/io/flutter/app/.
```

File: src/docs/release/breaking-changes/cupertino-icons-1.0.0.md [hunk 4]
```diff
@@ -98,26 +113,25 @@ dependencies:
...
+migrating to ensure that the automatically mapped
+new icons are suitable for your desired aesthetics.
 
 ## Timeline
 
 Landed in: 1.22.0-10.0.pre.65<br>
-In stable release: not yet
+In stable release: 1.22
 
 ## References
```
Comment (kwalrath) [inline src/docs/release/breaking-changes/cupertino-icons-1.0.0.md:133]:
```suggestion
```

File: src/docs/release/breaking-changes/material-chip-button-semantics.md [hunk 5]
```diff
@@ -131,13 +146,18 @@ API documentation:
 * [`SemanticsProperties.button`][]
 * [`SemanticsProperties.enabled`][]
 
-Relevant issues:
-* [flutter/flutter#58010][]
+Relevant issue:
+
+* [Issue 58010][]: InputChip doesn't announce any
```
Comment (kwalrath) [inline src/docs/release/breaking-changes/material-chip-button-semantics.md:151]: Link not defined

File: src/docs/release/breaking-changes/material-chip-button-semantics.md [hunk 6]
```diff
@@ -131,13 +146,18 @@ API documentation:
...
+  action for a11y on iOS
 
 Relevant PRs:
-* [Tweaking Material Chip a11y semantics to match buttons][]
-* [Revert "Tweaking Material Chip a11y semantics to match buttons (#60141)"][]
-* [Re-land "Tweaking Material Chip a11y semantics to match buttons (#60141)"][]
+
+* [PR 60141][]: Tweaking Material Chip a11y semantics
+  to match buttons][]
```
Comment (kwalrath) [inline src/docs/release/breaking-changes/material-chip-button-semantics.md:157]:
```suggestion
  to match buttons
```

**Review by kwalrath — APPROVED** [review]
I did a fairly quick review, and noticed a few issues with links

File: src/docs/release/breaking-changes/material-chip-button-semantics.md (hunk 5 above)
Comment (sfshaza2) [reply src/docs/release/breaking-changes/material-chip-button-semantics.md:151]: I'll fix in a follow up.

**Comment by deandreamatias** [comment]
Important to say to add that this PR solve the issue #4791

**Pull request merged**